Recipes/definitions.recipe.json

```diff
@@ -2600,11 +2586,11 @@
               "Name": "MenuToggleButtonLabel",
               "Settings": {
                 "ContentPartFieldSettings": {
-                  "DisplayName": "Menu Toggle Button Label",
+                  "DisplayName": "Menu Open Button Label",

```

Comment:
Did you mean to change this as well?

Reply:
Yes, it was another change that got reverted and I figured I might as well fix it while I was here - we have Open and Close labels now, neither should be called 'Toggle'.